lib/features/composer/domain/usecases/send_email_interactor.dart

```diff
@@ -36,10 +41,22 @@ class SendEmailInteractor {
       final currentMailboxState = listState.first;
       final currentEmailState = listState.last;
 
+      bool alwaysReadReceipt = true;
+      EmailRequest? updatedEmailRequest;
+      try {
+        final serverSettings = await _serverSettingsRepository.getServerSettings(accountId);
+        alwaysReadReceipt = serverSettings.settings?.alwaysReadReceipts ?? true;
+      } catch (_) {
+        alwaysReadReceipt = true;
+      }
```

Comment:
Should be separated into separate functions for processing

Reply:
done